luxtronik/__main__.py

```diff
@@ -0,0 +1,37 @@
+"""Luxtronik CLI."""
+import argparse
+import sys
+from luxtronik.discover import discover as _discover
+from luxtronik.scripts.dump import dump  # pylint: disable=unused-import # noqa: F401
+
+
+def discover():
+    """Nicely printing discovery results."""
+    results = _discover()
+    print(f"{len(results)} heatpump(s) reported back")
+    for result in results:
+        print(f"IP Address: {result[0]} Port: {result[1]}")
```

Comment:
I would suggest to add some counter here:

```suggestion
    for count, result in enumerate(results):
        print(f"Heat pump #{count} -> IP address: {result[0]} port: {result[1]}")
```